src/main/java/com/google/api/codegen/DiscoveryImporter.java

```diff
@@ -41,7 +41,7 @@
  * DiscoveryImporter parses a Discovery Doc into a {@link com.google.api.Service} object and an
  * additional {@link ApiaryConfig} object for data not accommodated by Service.
  *
- * The current implementation also provides a main method used for manual sanity checking.
+ * <p>The current implementation also provides a main method used for manual sanity checking.
```

Comment:
I think it happens when I run google-java-format. It suspect it applies changes that are not mandated by the verifier.